### PR TITLE
Ft user create profile 167165060

### PR DESCRIPTION
**Pull request by PelumiAlesh**
### What does this PR do?
Create endpoint for user to create, edit and view profile
#### Description of Task to be completed?
Create the endpoint for the following
> User can create profile
> User can edit their own profile
> User can view their own profile and others profile
#### How should this be manually tested?
After cloning 
Run `npm run server`
Then make a `POST` and `PATCH` request to /api/v1/profile . To view a profile make a `GET` request to /api/v1/profile/:id
#### What are the relevant pivotal tracker stories?
#167165060 
#### Screenshots (if appropriate)

Screenshot 2019-07-27 at 5 52 29 PM

### CODE REVIEW

**Review by mekzy-o — CHANGES_REQUESTED** [review]
Nice one Pelumi.. But u need to rebase from develop, and rectify the signup conflicts, I changed few things. Also check your hound violations

File: .eslintrc [hunk 1]
```diff
@@ -12,7 +12,6 @@
     "new-cap": 0,
     "consistent-return": 0,
     "no-param-reassign": 0,
-    "comma-dangle": ["error", "only-multiline"],
```
Comment (tolulope-od) [inline .eslintrc:15]: Please, why was this rule removed?

File: controllers/AuthController.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default class AuthController {
       firstName, lastName, email, password,
     } = req.body;
 
-    const foundUser = await User.findOne({ where: { email } });
```
Comment (tolulope-od) [inline controllers/AuthController.js:31]: Hey, is there any reason why this destructuring was removed?

**Review by tolulope-od — COMMENTED** [review]
Great work here @PelumiAlesh Could you please fix the hound errors? Thanks!

**Pull request closed without merging**